deploy/central/tasks-chart/templates/task-ods-build-typescript-with-sidecar.yaml

```diff
@@ -5,52 +5,31 @@ metadata:
   creationTimestamp: null
   name: '{{default "ods" .Values.taskPrefix}}-build-typescript-with-sidecar{{.Values.taskSuffix}}'
 spec:
-  description: |-
-    Builds Typescript applications.
-
-    The following steps are executed:
-
-    - linting using `eslint`
-    - build typescript application, using `npm run build`
-    - test execution
-    - SonarQube quality scan
-
-    For `eslint` to work there needs to be a config file (`eslintrc.json` or similar) at the root of the working directory.
-    This can be done by running `eslint --init` or by following the link:https://eslint.org/docs/user-guide/getting-started[official documentation]
-
-    The exact build recipe can be found at
-    link:https://github.com/opendevstack/ods-pipeline/blob/master/build/package/scripts/build-typescript.sh[build/package/scripts/build-typescript.sh].
-    In particular, `npm run build` is expected to place outputs into `dist`.
-
-    After tests ran successfully, the application source code is scanned by SonarQube.
-    Default SonarQube project properties are provided unless `sonar-project.properties`
-    is present.
-    When `sonar-quality-gate` is set to `true`, the task will fail if the quality gate
-    is not passed. If SonarQube is not desired, it can be disabled via `sonar-skip`.
-    The SonarQube scan will include parameters to perform a pull request analysis if
-    there is an open pull request for the branch being built. If the
-    link:https://docs.sonarqube.org/latest/analysis/bitbucket-integration/[ALM integration]
-    is setup properly, pull request decoration in Bitbucket is done automatically.
-
-    The following artifacts are generated by the build task and placed into `.ods/artifacts/`
-
-    * `code-coverage/`
-      ** `clover.xml`
-      ** `coverage-final.json`
-      ** `lcov.info`
-    * `lint-reports`
-      ** `report.txt`
-    * `sonarqube-analysis/`
-      ** `analysis-report.md`
-      ** `issues-report.csv`
-      ** `quality-gate.json`
-    * `xunit-reports/`
-      ** `report.xml`
-
-    **Sidecar variant!** Use this task if you need to run a container next to the build task.
-    For example, this could be used to run a database to allow for integration tests.
-    The sidecar image to must be supplied via `sidecar-image`.
-    Apart from the sidecar, the task is an exact copy of `ods-build-typescript`.
+  description: "Builds Typescript applications.\n\nThe following steps are executed:\n\n-
```

Comment:
did you generate this using the `make` target? It's odd that the description contains `\n` ...

Reply:
Yes I used make as it was indicated when the build failed. I was a bit surprised as well, but thought perhaps somebody manually formatted before and that is why it looks different. I did however not yet check into how other generated files look like.
BTW the make commands to update the sidecars and then the docs would be helpful to have mentioned perhaps in the development docs? The checks only run towards the end of the build and so it wastes quite some time if one forgets to do this locally.